pkg/kubelet/config/config.go

```diff
@@ -265,6 +272,7 @@ func (s *podStorage) merge(source string, change interface{}) (adds, updates, de
 				ref.Annotations = make(map[string]string)
 			}
 			ref.Annotations[kubelet.ConfigSourceAnnotationKey] = source
+			recordFirstSeenTime(ref)
```

Comment:
you don't need this and start time right (if we decide to set startime as watch received time)? Can't you just set it on the pod when it's added and everything will just work (https://github.com/GoogleCloudPlatform/kubernetes/pull/10066 shouldn't overwrite it)